addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -8,15 +8,42 @@
     justify-content: space-between;
 }
 
+.o_ChatterTopbar_buttonAttachmentsCountLoader {
+    margin-left: 2px;
+}
+
 .o_ChatterTopbar_buttonCount {
-    padding-left: 0.25rem !important;
+    padding-left: 0.25rem;
 }
 
+.o_ChatterTopbar_buttonSendMessage {
+    padding: 0 16px;
+    border-left: none;
+
+    &.o-buttonActive {
+        border-left: none;
+    }
+}
+
+.o_ChatterTopbar_leftSection {
+    display: flex;
+}
 
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
+.o_ChatterTopbar {
+
+    .o-buttonActive {
+        color: #875A7B;
+        background-color: lighten(gray('300'), 7%);
+        border-color: #dee2e6;
```

Comment:
why no use `gray()` variant?

Reply:
I think I reused the initial style or maybe I copied it from chrome inspector, I'll check that

Reply:
How would you rewrite it ?